apps/www/src/tests/factories/feature.factory.ts

```diff
@@ -1,15 +1,26 @@
 import {Feature} from '@/models/feature.model'
 import {faker} from '@faker-js/faker'
 import {iconNames, iconTypes} from '@/models/icon.model'
+import {imageFactory} from './image.factory'
 
 export type FeatureFactoryProps = {
-  type: (typeof iconTypes)[number]
+  useImage?: boolean
```

Comment:
Instead of `useImage: boolean` -> `type: 'icon' | 'image'` and so 
`type?: (typeof iconTypes)[number]` -> `iconType?: (typeof iconTypes)[number]`